pyston/serializer.py

```diff
@@ -336,6 +340,13 @@ def _get_m2m_fields(self, obj):
                 out[mf.name] = mf
         return out
 
+    def _get_reverse_fields(self, obj):
+        out = {}
+        for rf in obj._meta.get_fields():
+            if (rf.one_to_many or rf.one_to_one) and rf.auto_created and not rf.concrete:
+                out[rf.name] = rf
+        return out
```

Comment:
I need reverse fieds because getattr(obj, field_name) raises AttributeError if related object does not exists